plugin-server/src/worker/ingestion/process-event.ts

```diff
@@ -670,8 +794,8 @@ export class EventsProcessor {
         if (isNewPerson) {
             // Catch race condition where in between getting and creating, another request already created this user
             try {
-                await this.db.createPerson(
-                    sentAt,
+                await this.createPerson(
+                    DateTime.utc(),
```

Comment:
I assume it should always be when we created the person? It's used differently in different places

Reply:
Well no. Let's use the event timestamp here. It's a confusing name and elsewhere `created_at` is indeed when we created something. But think about what we display in the UI: We tell people this person was `first seen x months ago`. If I migrate events, I'll get an incorrect reading. 

This is a tricky one as it does end up being order-dependent though 🤔 maybe we need to talk about this further afterwards. essentially, with the way we display things, we create the expectation that `created_at == first event timestamp`, which is not true. It's also not something that is so important that we want to rework ingestion for this (e.g. try to update the person on every event).